gnupg.py

```diff
@@ -1546,6 +1576,32 @@ def gen_key_input(self, **kwargs):
         # %secring foo.sec
         # %commit
 
+    def addSubKey(self, master_key, master_passphrase=None, 
```

Comment:
I've generally used snake_case for method names rather than camelCase, so please change this to `add_subkey()` everywhere for consistency.

Reply:
Should I cange the class name to snake_case too?

Reply:
Note that the _class name_ `AddSubKey` is fine, it conforms to Python (and this project's) conventions.